CHANGES.md

```diff
@@ -12,6 +12,7 @@ Fixed Issues:
 
 * [#2672](https://github.com/ckeditor/ckeditor-dev/issues/2672): Fixed: When resizing [Enhanced Image](https://ckeditor.com/cke4/addon/image2) to minimum size with a resizer the image dialog doesn't show actual values.
 * [#1478](https://github.com/ckeditor/ckeditor-dev/issues/1478): Fixed: Custom colors added to [Color Button](https://ckeditor.com/cke4/addon/colorbutton) via [`config.colorButton_colors`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-colorButton_colors) in form label/code don't work correctly.
+* [#2813](https://github.com/ckeditor/ckeditor-dev/issues/2813): Fixed: [editor.insertHtml](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-insertHtml) and [editor.insertelement](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-insertElement) pollutes editable with empty spans.
```

Comment:
PR also covers `insertElementIntoRange` and `insertHtmlIntoRange`. Maybe instead of listing method names, we could make it more general, like:

`Fixed: Editor HTML insertion methods pollute the content with empty spans.`?